SMTInterpolTest/src/de/uni_freiburg/informatik/ultimate/smtinterpol/smtlib2/APITest.java

```diff
@@ -369,7 +369,7 @@ public static void randomWalk(SMTInterpol solver, int n) {
 	}
 
 	@Test
-	public void testPreproducibleResourceLimit() {
+	public void testRreproducibleResourceLimit() {
```

Comment:
Still an r too much :) also in the other test.

Reply:
Should be good this time.
I think I may need another coffee.